multicast/__main__.py

```
@@ -626,18 +626,18 @@
+		except Exception as _cause:  # pragma: no branch
+			exit_code = exceptions.get_exit_code_from_exception(_cause)
 			__EXIT_MSG = (
-				False,
-				f"CRITICAL - Attempted '[{__name__}]: {args}' just failed! :: {exit_code} {err}" # noqa
+				1,
+				f"CRITICAL - Attempted '[{__name__}]: {args}' just failed! :: {exit_code} {_cause}" # noqa
 			)
 			if (sys.stderr.isatty()):
 				print(
 					"WARNING - An error occurred while handling the arguments. Refused.",
 					file=sys.stderr
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)